clients/src/main/java/org/apache/kafka/clients/FetchSessionHandler.java

```diff
@@ -377,6 +380,7 @@ private String responseDataToLogString(FetchResponse<?> response) {
      *                  because of missing or unexpected partitions.
      */
     public boolean handleResponse(FetchResponse<?> response) {
+        boolean success;
         if (response.error() != Errors.NONE) {
             log.info("Node {} was unable to process the fetch request with {}: {}.",
```

Comment:
Is warning level more appropriate than info? Especially since we no longer expect the `INVALID_FETCH_SESSION_EPOCH` error to commonly occur.